src/components/ConfirmationButton.tsx

```diff
@@ -0,0 +1,44 @@
+import noop from 'lodash.noop';
+import type { FC, MouseEventHandler, ReactNode } from 'react';
+import React, { useCallback, useState } from 'react';
+import type { ButtonProps } from './Button';
+import Button from './Button';
+
+export interface ConfirmationButtonProps extends ButtonProps {
+  children?: ReactNode;
```

Comment:
I don't think `children` is needed here

Reply:
Oh man. That one got me again. Thanks for catching!